### PR TITLE
[Console] Escape question text and default value in SymfonyStyle::ask()

### PR DESCRIPTION
**Pull request by chalasr**
| Q | A |
| --- | --- |
| Branch? | 2.7 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #19493 |
| License | MIT |
| Doc PR | n/a |

### CODE REVIEW

**Comment by ro0NL** [comment]
This needs to be done on a per case basis? Ie. `case $question instanceof ChoiceQuestion:` requires a different approach right?

File: src/Symfony/Component/Console/Helper/SymfonyQuestionHelper.php [hunk 1]
```diff
@@ -52,9 +53,13 @@ public function ask(InputInterface $input, OutputInterface $output, Question $qu
      */
     protected function writePrompt(OutputInterface $output, Question $question)
     {
-        $text = $question->getQuestion();
+        $text = OutputFormatter::escape($question->getQuestion());
         $default = $question->getDefault();
 
+        if (null !== $default) {
```
Comment (sstok) [inline src/Symfony/Component/Console/Helper/SymfonyQuestionHelper.php]: Needs to be skipped for `ConfirmationQuestion`, else bool is converted to a string.

**Review by sstok — CHANGES_REQUESTED** [review]

**Comment by chalasr** [comment]
I made the change, refactoring a bit btw so we stop duplications.
If it looks less readable, please say me so I rollback and do the change for each case of the switch that need it.

**Comment by ro0NL** [comment]
Yeah.. i would say revert and escape where needed.

**Comment by chalasr** [comment]
Reverted.

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**